rs-commerce/src/components/login-page-form/style.module.css

```diff
@@ -0,0 +1,91 @@
+.login-page-form {
+  padding: 36px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+.login-page-form-title {
+  font-family: var(--font-poppins);
+  font-weight: bold;
+  font-size: 48px;
+  color: #333;
+}
+.login-page-form-register {
+  font-family: var(--font-poppins);
+  font-weight: bold;
+  font-size: 16px;
+  color: #333;
+}
+.text-blue {
+  color: #209fc1;
+  cursor: pointer;
+  transition: var(--transition-time);
+}
+.text-blue:hover {
+  color: #eb5798;
+}
+.input-wrapper {
+  width: 400px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+}
+.password__eye {
+  position: absolute;
+  top: 48px;
+  right: 28px;
+  width: 24px;
+  height: 24px;
+  background: transparent url(../../assets/login-page/eye_closed.svg) center center no-repeat;
+  cursor: pointer;
+  transition: var(--transition-time);
+}
+.eye-open {
+  background: transparent url(../../assets//login-page/eye_open.svg) center center no-repeat;
```

Comment:
`../../assets//login-page/eye_open.svg` Does it work well with two slashes?

Reply:
I corrected the typo, but it worked anyway